docs/UserGuide.adoc

```diff
@@ -264,20 +264,43 @@ Deletes a person with the NRIC S98654322D in ORGANice.
 // end::delete[]
 === Detecting matches: `match`
 
-Currently, ORGANice only supports kidney matching. Refer to <<FAQ>> to know the
+Currently, ORGANice only supports kidney matching. Refer to <<FAQ>> for a better understanding of the
 procedure for kidney matching.
 
 This command displays a list of patient-donor pairs that passes both the blood type and tissue
 typing tests. +
 
-To get a list of patients and their number of matches, the format is: `match ic/all`
+There are two types of match commands. The following list explains each type of match command and describes the result
+of executing each type of command:
 
-To get a list of potential matches a patient has with a donor, the format is: `match ic/**NRIC**`
+* To get a list of patients and their number of matches, the format is: `match ic/all`
```

Comment:
Perhaps
To get a list of all patients with the number of matches they have respectively 
Might be better to understan?

Reply:
Potential matches*